input/samples/vn_simona/vn_simona_minimal.conf

```diff
@@ -0,0 +1,104 @@
+# beginning of your config file and then just override the parameters you want to change!
+#########
+
+##################################################################
+# Simulation Parameters
+##################################################################
+simona.simulationName = "vn_simona"
+
+##################################################################
+# Time Parameters
+##################################################################
+simona.time.startDateTime = "2011-01-01T00:00:00Z"
+simona.time.endDateTime = "2011-01-01T02:00:00Z"
+simona.time.schedulerReadyCheckWindow = 900
+
+##################################################################
+# Input Parameters
+##################################################################
+simona.input.primary.csvParams = {
+  directoryPath: "input/samples/vn_simona/fullGrid"
+  csvSep: ","
+  isHierarchic: false
+}
+simona.input.grid.datasource.id = "csv"
+simona.input.grid.datasource.csvParams = {
+  directoryPath: "input/samples/vn_simona/fullGrid"
+  csvSep: ","
+  isHierarchic: false
+}
+
+simona.input.weather.datasource = {
+  sampleParams.use = true
+  coordinateSource.sampleParams.use = true
+}
+
+##################################################################
+# Output Parameters
+##################################################################
+simona.output.base.dir = "output/vn_simona"
+simona.output.sink.csv.csvSep = ","
+
+simona.output.participant.defaultConfig.simulationResult = true
+simona.output.participant.individualConfigs = [
+  {
+    notifier = "pv"
+    simulationResult = true
+  },
+  {
+    notifier = "wec"
+    simulationResult = true
+  },
+  {
+    notifier = "evcs"
+    simulationResult = true
+  }
+]
+simona.output.thermal.individualConfigs = [
+  {
+    notifier = "house",
+    simulationResult = true
+  },
+  {
+    notifier = "cylindricalstorage",
+    simulationResult = true
+  }
+]
+
+##################################################################
+# Runtime Configuration // todo refactor as this naming is misleading and partly unneeded
+##################################################################
+
+##################################################################
+# Grid Configuration
+##################################################################
+
+##################################################################
+# Power Flow Configuration
+##################################################################
+simona.powerflow.newtonraphson.epsilon = [1E-12]
+simona.powerflow.stopOnFailure = true
+
+simona.control.transformer = [
+  {
+    transformers = ["31a2b9bf-e785-4475-aa44-1c34646e8c79"],
+    measurements = ["923f2d69-3093-4198-86e4-13d2d1c220f8"],
+    vMin = 0.98,
+    vMax = 1.02
+  }, {
+    transformers = ["1132dbf4-e8a1-44ae-8415-f42d4497aa1d"],
+    measurements = ["7686b818-a0ba-465c-8e4e-f7d3c4e171fc"],
+    vMin = 0.98,
+    vMax = 1.02
+  }
+]
```

Comment:
General question as well for 'vn_simona.conf': Is this realy part of the Power Flow Conf?